react-props-and-expressions/my-app/src/App.jsx

```diff
@@ -0,0 +1,14 @@
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  return (
+    <div>
+      <CustomButton text={'I'} color="red" />
+      <CustomButton text={'know'} color="blue" />
+      <CustomButton text={'React!'} color="green" />
```

Comment:
Works, but since you're not writing JavaScript and just assigning a string, you can just do `text="I"` directly.

We only include those curly braces when we want to explicitly write JS